### DISCUSSION TITLE
No longer possible to convert an issue to a discussion?

### DISCUSSION THREAD

**Discussion by michaelklishin** [community/community#3115] [Discussions] [answered]
Looks like the feature described in #2952 was removed or unintentionally hidden from the UI. I can no longer find the button to convert a "question issue" to a discussion.

This has largely eliminated the benefit of having discussions for my team: we routinely see users file issues for what are questions or requests for troubleshooting help, no issues with any amount of evidence. We would typically convert those to discussions but no longer have this option 😭. Now we have to transfer such issues to a separate repository which makes them significantly more difficult to discover for others. This increases the time cost of maintaining open source software for us.

**Comment by zayfod** [comment, marked as answer]
See https://github.com/github/feedback/discussions/3105

**Reply by michaelklishin** [reply]
Understood, thank you for pointing this out. #3105 has the context that explains that the feature was temporarily disabled before some known bugs around it are addressed 👍
